src/network/networkd-dhcp6.c

```diff
@@ -405,21 +405,10 @@ static int dhcp6_address_handler(sd_netlink *rtnl, sd_netlink_message *m, Link *
 
         r = sd_netlink_message_get_errno(m);
         if (r < 0 && r != -EEXIST) {
-                if (link->rtnl_extended_attrs) {
```

Comment:
Yes, I think it's reasonable to drop this. I hope people are not using such for old kernels for networking with ipv6.